// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`.not.toLog fails when the callback function outputs the expected message 1`] = `
"<dim>expect(</><red>received</><dim>).not.toLog(</><green>expected</><dim>)</>

Expected console.log not to have been called with:
  <green>\\"console-log\\"</>
But it was called with:
  <red>[[\\"console-log\\"]]</>"
`;

exports[`.not.toLog fails when the callback function outputs the expected message 2`] = `
"<dim>expect(</><red>received</><dim>).not.toLog(</><green>expected</><dim>)</>

Expected console.warn not to have been called with:
  <green>\\"console-warn\\"</>
But it was called with:
  <red>[[\\"console-warn\\"]]</>"
`;

exports[`.not.toLog fails when the callback function outputs the expected message 3`] = `
"<dim>expect(</><red>received</><dim>).not.toLog(</><green>expected</><dim>)</>

Expected console.error not to have been called with:
  <green>\\"console-error\\"</>
But it was called with:
  <red>[[\\"console-error\\"]]</>"
`;

exports[`.not.toLog fails when the callback function outputs the expected message 4`] = `
"<dim>expect(</><red>received</><dim>).not.toLog(</><green>expected</><dim>)</>

Expected console.info not to have been called with:
  <green>\\"console-info\\"</>
But it was called with:
  <red>[[\\"console-info\\"]]</>"
`;

exports[`.not.toLog fails when the callback function outputs the expected message 5`] = `
"<dim>expect(</><red>received</><dim>).not.toLog(</><green>expected</><dim>)</>

Expected console.assert not to have been called with:
  <green>\\"console-assert\\"</>
But it was called with:
  <red>[[\\"console-assert\\"]]</>"
`;

exports[`.toLog fails when the callback function does not output any message 1`] = `
"<dim>expect(</><red>received</><dim>).toLog(</><green>expected</><dim>)</>

Expected console.log to have been called with:
  <green>\\"something\\"</>
But it was not called"
`;

exports[`.toLog fails when the callback function does not output any message 2`] = `
"<dim>expect(</><red>received</><dim>).toLog(</><green>expected</><dim>)</>

Expected console.warn to have been called with:
  <green>\\"something\\"</>
But it was not called"
`;

exports[`.toLog fails when the callback function does not output any message 3`] = `
"<dim>expect(</><red>received</><dim>).toLog(</><green>expected</><dim>)</>

Expected console.error to have been called with:
  <green>\\"something\\"</>
But it was not called"
`;

exports[`.toLog fails when the callback function does not output any message 4`] = `
"<dim>expect(</><red>received</><dim>).toLog(</><green>expected</><dim>)</>

Expected console.info to have been called with:
  <green>\\"something\\"</>
But it was not called"
`;

exports[`.toLog fails when the callback function does not output any message 5`] = `
"<dim>expect(</><red>received</><dim>).toLog(</><green>expected</><dim>)</>

Expected console.assert to have been called with:
  <green>\\"something\\"</>
But it was not called"
`;

exports[`.toLog fails when the callback function does not output the expected message 1`] = `
"<dim>expect(</><red>received</><dim>).toLog(</><green>expected</><dim>)</>

Expected console.log to have been called with:
  <green>\\"something\\"</>
But it was called with:
  <red>[[\\"console-log\\"]]</>"
`;

exports[`.toLog fails when the callback function does not output the expected message 2`] = `
"<dim>expect(</><red>received</><dim>).toLog(</><green>expected</><dim>)</>

Expected console.warn to have been called with:
  <green>\\"something\\"</>
But it was called with:
  <red>[[\\"console-warn\\"]]</>"
`;

exports[`.toLog fails when the callback function does not output the expected message 3`] = `
"<dim>expect(</><red>received</><dim>).toLog(</><green>expected</><dim>)</>

Expected console.error to have been called with:
  <green>\\"something\\"</>
But it was called with:
  <red>[[\\"console-error\\"]]</>"
`;

exports[`.toLog fails when the callback function does not output the expected message 4`] = `
"<dim>expect(</><red>received</><dim>).toLog(</><green>expected</><dim>)</>

Expected console.info to have been called with:
  <green>\\"something\\"</>
But it was called with:
  <red>[[\\"console-info\\"]]</>"
`;

exports[`.toLog fails when the callback function does not output the expected message 5`] = `
"<dim>expect(</><red>received</><dim>).toLog(</><green>expected</><dim>)</>

Expected console.assert to have been called with:
  <green>\\"something\\"</>
But it was called with:
  <red>[[\\"console-assert\\"]]</>"
`;

exports[`.toLog fails when the callback function does not output the expected message 6`] = `
"<dim>expect(</><red>received</><dim>).toLog(</><green>expected</><dim>)</>

Expected console.log to have been called with:
  <green>\\"something\\"</>
But it was called with:
  <red>[[\\"a\\", \\"b\\", \\"c\\"], [\\"console-log\\"]]</>"
`;
